model/task_lifecycle.go

```diff
@@ -1203,8 +1282,18 @@ func evalLinearStepback(ctx context.Context, t *task.Task, caller, status string
 
 // evalBisectStepback performs bisect stepback on the task.
 func evalBisectStepback(ctx context.Context, t *task.Task, caller, status string, deactivatePrevious bool) error {
-	// TODO: EVG-20788 implement stepback bisection.
-	return errors.Wrap(doBisectStepback(ctx, t), "performing bisect stepback")
+	// If the task is aborted or it isn't a mainline commit or existing stepback, no-op.
+	if t.Aborted || (t.ActivatedBy != evergreen.RepotrackerVersionRequester && t.ActivatedBy != evergreen.StepbackTaskActivator) {
+		return nil
+	}
+
```

Comment:
Can we add a brief comment explaining these two variables for future readability?